src/Types/Resources.ts

```diff
@@ -0,0 +1,27 @@
+export interface Resources {
+  label: string; // Name to be displayed to the user
+  name: string; // unique id for reference
+  description: string; // '' for null
+  tags: string[]; // [] for none
```

Comment:
Tags should reference a tag id and not the string itself, only because we want to ensure uniformity for tags.

Reply:
We decided to take the tags out of Resources